tareas/tarea-6/tarea-clase-6-k2.html

```diff
@@ -0,0 +1,150 @@
+<!--
+TAREA:
+Crear una interfaz que permita agregar ó quitar (botones agregar y quitar) inputs+labels para completar el salario anual de cada integrante de la familia que trabaje.
+Al hacer click en "calcular", mostrar en un elemento pre-existente el mayor salario anual, menor salario anual, salario anual promedio y salario mensual promedio.
+
+Punto bonus: si hay inputs vacíos, ignorarlos en el cálculo (no contarlos como 0).
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>K</title>
+</head>
+<style>
+    .botones{
+        display: flex;
+        justify-content: center;
+    }
+    .botones > input{
+        margin: 10px;
+    }
+    #divisores{
+        display: flex;
+    }
+    #todo{
+        text-align:center;
+        float: right;
+    }
+    .Datos{
+        float: left;
+    }
+    ul > *{
+        justify-content: center;
+        text-align:center
+    }
+</style>
+<body>
+    <div id="divisor">
+        <div class="Datos">
+            <h1>El que mas gana: </h1>
+            <h2>El promedio anul es : </h2>
+            <h2>El promedio mensual es : </h2>
+            <h3>El que menos gana: </h3>
+        </div>
+        <form id="todo">
+            <ul id="#formulario">
+                <label>Integrantes de la family</label>
+                <input id="#family" type="number" min ="0">
+            </ul>
+        <div class="botones">
+            <input type="button" id="submit" value="Calcular">
+            <input type="button" id="delete-this" value="Borra todo">
+        </div>
+        </form>
+    </div>
+</body>
+<script>
+    let cantidadVieja = 0;
+    let cantidadActual = 0;
+    const integrantes = document.getElementById("#family");
+    const borrarTodo = document.getElementById("delete-this");
+    const calcular = document.getElementById("submit"); 
+    borrarTodo.addEventListener("click",function(e){
+        let i = 0;
+        while(cantidadActual>=i){ 
+                del();
+                cantidadActual--;
+        }
+        integrantes.value = 0;
+        modify_data("","","");
+    })
+    integrantes.addEventListener("change",function(e){
+        cantidadActual = integrantes.value;
+        if(cantidadActual>cantidadVieja){
+            while(cantidadVieja!=cantidadActual){ 
+                add();
+                cantidadVieja++;
+            }
+        }else{
+            while(cantidadVieja!=cantidadActual){ 
+                del();
+                cantidadVieja--;
+            }
+        } 
+    })
+
+    function add(){
+        const l = document.createElement("label");
```

Comment:
En un año vas a leer esa `l` y vas a preguntar qué estabas fumando.

```suggestion
        const $label = document.createElement("label");
```